spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/otlp/OtlpAutoConfigurationIntegrationTests.java

```diff
@@ -113,4 +141,88 @@ void httpSpanExporterCanBeConfiguredToUseGzipCompression() {
 			});
 	}
 
+	@Test
+	void grpcSpanExporter() {
+		this.contextRunner
+			.withPropertyValues(
+					"management.otlp.tracing.endpoint=http://localhost:%d".formatted(this.mockGrpcServer.getPort()),
```

Comment:
`http://`? Should not this be just `localhost:<port>`?
It's interesting that the grpc exporter still works if the protocol scheme is http (even though it is using http under the hood).

Reply:
I'm afraid that's not an option. The `OltpGrpcSpanExporterBuilder` actually validates that the URL starts with `http://` or `https://`.

```
Caused by: java.lang.IllegalArgumentException: Invalid endpoint, must start with http:// or https://: localhost:49192
	at io.opentelemetry.exporter.internal.ExporterBuilderUtil.validateEndpoint(ExporterBuilderUtil.java:35)
	at io.opentelemetry.exporter.internal.grpc.GrpcExporterBuilder.setEndpoint(GrpcExporterBuilder.java:104)
	at io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporterBuilder.setEndpoint(OtlpGrpcSpanExporterBuilder.java:130)
	at org.springframework.boot.actuate.autoconfigure.tracing.otlp.OtlpTracingConfigurations$Exporters.otlpGrpcSpanExporter(OtlpTracingConfigurations.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:140)
	... 106 more
```